static/src/stylesheets/module/facia-garnett/_item.scss

```diff
@@ -186,7 +186,17 @@ $fc-item-gutter: $gs-gutter / 4;
     position: relative;
     padding-right: 4px;
```

Comment:
we shoudln't really be using numbers liek this, we don't elsewhere. is there a reason to avoid `$gs-gutter`/`$gs-baseline`?

Reply:
Yeah these rules are around the kicker's spacing. I can derive all these values from `$gs-gutter/$gs-baseline`, but I wasn't convinced these rules should be tied to `$gs-gutter/$gs-baseline`. If `$gs-gutter/$gs-baseline` were to change we wouldn't necessarily want these properties to adjust accordingly. I'll swap them out with references to those variables and see how it behaves...